### PR TITLE
Rough draft of IPFS Weekly #16

### PR DESCRIPTION
**Pull request by jennwrites**
Looking for feedback and suggestions on this draft so that we can send it out via TinyLetter (and post on the blog) Tuesday, October 30. 🎃

Thanks!

### CODE REVIEW

**Comment by pkafei** [comment]
This is looking good, thank you Jenn! The only thing I would add is, I would mention [devcon](https://devcon4.ethereum.org/) in the **Coming up in the Community** section. Also, I will try to figure out why that Travis test is failing.

**Comment by jamiew** [comment]
Looking great. I'd vote to put the [pre-registration for IPFS Conf 2019](https://docs.google.com/forms/d/e/1FAIpQLSfJVVPwvp6RY3MUg1zAVl1g_5y2nGb7WJIMI1Hs6glzm7FLHQ/viewform) higher up so people don't miss it (unless you're intentionally keeping it lower-key)

File: published/016-2018-oct-30.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
++ [Manyverse and Scuttlebutt: A human-centric technology stack for social applications](https://www.zdnet.com/article/manyverse-and-scuttlebutt-a-human-centric-technology-stack-for-social-applications/), 25 Oct 2018
++ [iPLD demo](https://twitter.com/jimpick/status/1055249910914633729), 24 Oct 2018
++ [YottaChain unlocks the potential of the IPFS protocol](http://www.digitaljournal.com/pr/3990415), 23 Oct 2018
++ [The 5 steps to end-to-end encrypted photo storage and sharing](https://medium.com/textileio/the-5-steps-to-end-to-end-encrypted-photo-storage-and-sharing-45ad4aad6b14) 23 Oct 2018
++ [PHOTO]: https://twitter.com/Josh0akley/status/1054346890651811840 22 Oct 2018
+
+## Updates and new releases
+*See the latest releases of IPFS tools and projects across the ecosystem.*
+
++
```
Comment (lidel) [inline published/016-2018-oct-30.md]: I think we could mention RC1 of go-ipfs:
```suggestion
+  [go-ipfs 0.4.18-rc1](https://github.com/ipfs/go-ipfs/blob/master/CHANGELOG.md#0418-2018-10-26-rc1) – First Release Candidate of largest go-ipfs release to date, 3 months in the making. If you are okay with living on the bleeding edge, please install and test! 
```

Comment (lidel) [reply published/016-2018-oct-30.md]: @Stebalien it sounds like a big release, would it help to get more testers?

Comment (momack2) [reply published/016-2018-oct-30.md]: Maybe also add some of the changes (experimental QUIC support, etc) so people know what to try out?

Comment (Stebalien) [reply published/016-2018-oct-30.md]: Yes, getting some more testers would be great!

**Comment by jesseclay** [comment]
@renrutnnej ooo past couple weeks have been all about conferences!

Web3 Summit:
Juan IPFS/ d-web:
https://web3summit.com/speaker/juan-benet/
https://www.youtube.com/watch?v=l44z35vabvA

Mike libp2p:
https://web3summit.com/speaker/mike-goelzer/
https://www.youtube.com/watch?v=xqVmEzsin3Y&list=PLxVihxZC42nHlc7J-UqeS-asagbehBeh3&index=30

Organic presentation on implementing payment channel with libp2p:
https://www.youtube.com/watch?v=t2qCmwUlY2Q

Also please be sure to highlight our efforts and presence at Devcon this week -- we'd love to meet and hang with as much of the ethereum community as we can. Will loop you in with our teammates there so they can surface activities & events they're up to.

File: published/016-2018-oct-30.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+## The latest
+
++ Check out the [new media section](https://libp2p.io/media/) of the libp2p site! 
++ [Sweet IPFS](https://discuss.ipfs.io/t/sweet-ipfs-full-ipfs-node-for-android/3779?u=lidel): Full IPFS node for Android
+
+## IPFS in the wild
+*Do you follow [IPFS on Twitter](https://twitter.com/IPFSbot)? For the latest mentions of IPFS in the news, check our Twitter feed or see the [latest articles on Awesome IPFS](https://awesome.ipfs.io/categories/articles/).* 
+
++ [Manyverse and Scuttlebutt: A human-centric technology stack for social applications](https://www.zdnet.com/article/manyverse-and-scuttlebutt-a-human-centric-technology-stack-for-social-applications/), 25 Oct 2018
++ [iPLD demo](https://twitter.com/jimpick/status/1055249910914633729), 24 Oct 2018
```
Comment (alanshaw) [inline published/016-2018-oct-30.md]: Typo - small "i"

**Comment by pkafei** [comment]
@mikeal @terichadbourne @alanshaw Does anyone know what `The command "git diff master.. --name-only | grep '.md' | xargs awesome_bot --allow-dupe --white-list https://github.com/ipfs/newsletter/issues/new" exited with 123.` Travis error means?

**Comment by terichadbourne** [comment]
Sorry @pkafei, no idea. I don't have much experience with Travis.

**Comment by jennwrites** [comment]
IPFS Weekly 16 was published:

On the [IPFS Weekly Updates issue](https://github.com/ipfs/ipfs/issues/151#issuecomment-434371335)
On the [IPFS blog](https://blog.ipfs.io/50-ipfs-weekly-16/)
And sent to 3041 subscribers [via TinyLetter](https://tinyletter.com/ipfsnewsletter/letters/ipfs-weekly-16)

**Comment by hsanjuan** [comment]
Should this be merged?

**Comment by pkafei** [comment]
@hsanjuan Looks like it was already merged https://github.com/ipfs/blog/commit/5439580ad701868b060c3f3a49d252502c9dd54a

**Pull request merged**